packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -111,10 +111,10 @@ const CloseButtonWrapper = isReactNative ? Box : Fragment;
 
 // todo: add all icons
```

Comment:
I think this should be an exhaustive list of components, remove the todo comment in that case?

Reply:
Fixed in https://github.com/razorpay/blade/pull/670/commits/acafaadda2ed45b4dadc991fe94dcab8dfb4cad9